sci-geosciences/josm-bin/metadata.xml

```diff
@@ -15,5 +15,6 @@
   </maintainer>
   <upstream>
     <changelog>https://josm.openstreetmap.de/wiki/StartupPage</changelog>
+    <remote-id type="github">JOSM/josm</remote-id>
```

Comment:
What is the benefit of filling that field? github is not where this project actually is hosted or developed, just a mirror.

So strictly speaking github is not upstream, just one downstream. If there is a good reason i think we should have a bug describing a problem that gets fixed, or at least some reasoning in the commit message.

Reply:
Interesting question. The project even uses subversion for source control, so you cannot submit PRs there. The ticketing system is trac, they disabled github issues entirely.

Just because upstream runs the subversion->git mirror, doesn't mean it's necessarily an asset that the package can make use of. I agree it would be wrong to add this metadata.